.github/driver.sh

```diff
@@ -24,6 +24,14 @@ test "$sysroot" = $desired_sysroot
     SYSROOT=/tmp RUSTFLAGS="--sysroot=$(rustc --print sysroot)" ../target/debug/cargo-clippy clippy --verbose
 )
 
+# Check that the --sysroot argument is only passed once via arg_file.txt (SYSROOT is ignored)
+(  
+    echo "fn main() {}" > target/driver_test.rs
+    echo "--sysroot="$(./target/debug/clippy-driver --print sysroot)"" > arg_file.txt
+    echo "--verbose" >> arg_file.txt
+    SYSROOT=/tmp ./target/debug/clippy-driver @arg_file.txt ./target/driver_test.rs
```

Comment:
Great test, but not the one I had in mind :sweat_smile:  Please keep it though, testing this is important, great catch!

Can you also add a test like in line 15 and 18, that tests that it is actually possible to set the sysroot via the arg_file, please?

Reply:
Yes. PTAL